src/mem.rs

```diff
@@ -2,9 +2,10 @@
 use crate::block::Block;
 use crate::cid::Cid;
 use crate::codec::Codec;
-use crate::error::{BlockNotFound, EmptyBatch, Result};
+use crate::error::{BlockNotFound, BlockTooLarge, EmptyBatch, Result};
 use crate::multihash::MultihashDigest;
 use crate::store::{AliasStore, ReadonlyStore, Store, StoreResult};
+use crate::MAX_BLOCK_SIZE;
```

Comment:
I'm still not a fan of having a hard-coded value in this crate, it should be possible to be set from the outside (application). Especially for the mem storage, I can see that you don't want to have this restriction.